api/diff.go

```diff
@@ -30,77 +30,72 @@ func apiDiffHandler(w http.ResponseWriter, r *http.Request) {
 	}
 }
 
-type diffResult struct {
-	Diff    shared.ResultsDiff `json:"diff"`
-	Renames map[string]string  `json:"renames,omitempty"`
+func loadDiffRuns(store shared.Datastore, q url.Values) (shared.TestRuns, error) {
+	if runIDs, err := shared.ParseRunIDsParam(q); err != nil {
```

Comment:
Don't think `run_id` param is documented anywhere in [api/diff](https://github.com/web-platform-tests/wpt.fyi/blob/master/api/README.md#apidiff). Would you mind doing a drive-by change on the README too?

Reply:
Also I think this is wrong - IIUC, if `ParseRunIDsParam` returns an error, you can still get `runs` through `runFilter` in line 58. This results in ignoring `filter` information

Reply:
Optional - Ignore when `err` is nil here and let `ParseTestRunFilterParams` (line 58) fetches the runs as a fallback?

Reply:
Good point! Done!